thirdeye/thirdeye-frontend/ember-cli-build.js

```diff
@@ -45,11 +45,23 @@ module.exports = function(defaults) {
     destDir: '/assets'
   });
 
-  app.import('bower_components/source-sans-pro/source-sans-pro.css')
+  app.import(app.bowerDirectory + '/source-sans-pro/source-sans-pro.css');
+  //ionRangeSlider assets - http://ionden.com/a/plugins/ion.rangeSlider/en.html
+  app.import('node_modules/ion-rangeslider/css/ion.rangeSlider.skinHTML5.css');
+  // app.import('node_modules/ion-rangeslider/img/sprite-skin-nice.png', {
```

Comment:
remove?

Reply:
No. This is not well documented on how to implement this code. So i left it as a nice to know. How to import images in a `ember-cli-build.js` file. Took half a day to found how to do it correctly.